### PR TITLE
Doobie

### PR DESCRIPTION
**Pull request by mlachkar**

### CODE REVIEW

File: core/src/main/scala/ch/epfl/scala/utils/Secret.scala [hunk 1]
```diff
@@ -0,0 +1,7 @@
+package ch.epfl.scala.utils
+
+case class Secret(private val value: String) extends AnyVal {
```
Comment (adpi2) [inline core/src/main/scala/ch/epfl/scala/utils/Secret.scala:3]: I don't think we will need that class.

File: core/src/main/scala/ch/epfl/scala/services/ScaladexDb.scala [hunk 2]
```diff
@@ -0,0 +1,8 @@
+package ch.epfl.scala.services
+
+import ch.epfl.scala.index.model.Project
+
+trait ScaladexDb {
```
Comment (adpi2) [inline core/src/main/scala/ch/epfl/scala/services/ScaladexDb.scala:5]:
```suggestion
trait Database {
```

File: core/src/main/scala/ch/epfl/scala/utils/Secret.scala (hunk 1 above)
Comment (mlachkar) [reply core/src/main/scala/ch/epfl/scala/utils/Secret.scala:3]: for password for the real database ?

File: core/src/main/scala/ch/epfl/scala/services/ScaladexDb.scala (hunk 2 above)
Comment (mlachkar) [reply core/src/main/scala/ch/epfl/scala/services/ScaladexDb.scala:5]: ok

**Pull request merged**